rbs/variables.tf

```diff
@@ -0,0 +1,105 @@
+#
+# General Vars
+#
+variable "dependencies" {
+  # See: https://github.com/hashicorp/terraform/issues/21418#issuecomment-495818852
+  type        = any
+  default     = []
+  description = "Work-around for Terraform 0.12's lack of support for 'depends_on' in custom modules."
+}
+variable "enable" {
+  type        = bool
+  description = "Enable flag for this module. If set to false, no resources will be created."
+  default     = true
+}
+variable "google_project" {
+  type        = string
+  description = "The google project in which to create resources"
+}
+variable "google_folder_id" {
+  type        = string
+  description = "The folder in which RBS has permission."
+  default     = ""
+}
+variable "cluster" {
+  type        = string
+  description = "Terra GKE cluster suffix, whatever is after terra-"
+}
+variable "cluster_short" {
+  type        = string
+  description = "Optional short cluster name"
+  default     = ""
+}
+variable "owner" {
+  type        = string
+  description = "Environment or developer. Defaults to TF workspace name if left blank."
+  default     = ""
+}
+locals {
+  owner   = var.owner == "" ? terraform.workspace : var.owner
+  service = "rbs"
+}
+
+#
+# DNS Vars
+#
+variable "dns_zone_name" {
+  type        = string
+  description = "DNS zone name"
+  default     = "dsp-envs"
+}
+variable "use_subdomain" {
+  type        = bool
+  description = "Whether to use a subdomain between the zone and hostname"
+  default     = true
+}
+variable "subdomain_name" {
+  type        = string
+  description = "Domain namespacing between zone and hostname"
+  default     = ""
+}
+variable "hostname" {
+  type        = string
+  description = "Service hostname"
+  default     = ""
+}
+locals {
+  hostname       = var.hostname
+  cluster_name   = var.cluster_short
+  subdomain_name = var.use_subdomain ? var.subdomain_name : ""
+}
+
+
+#
+# Postgres CloudSQL DB Vars
+#
+variable "db_tier" {
+  default     = "db-g1-small"
+  description = "The default tier (DB instance size) for the CloudSQL instance"
+}
+variable "db_name" {
+  type        = string
+  description = "Postgres db name"
+  default     = ""
+}
+variable "db_user" {
+  type        = string
+  description = "Postgres username"
+  default     = ""
+}
+variable "stairway_db_name" {
+  type        = string
+  description = "Stairway db name"
+  default     = ""
+}
+variable "stairway_db_user" {
+  type        = string
+  description = "Stairway db username"
+  default     = ""
+}
+locals {
+  db_name          = var.db_name
+  db_user          = var.db_user
+  stairway_db_name = var.stairway_db_name
+  stairway_db_user = var.stairway_db_user
```

Comment:
if we are going to use locals, they should default to something that's not an empty string.

Reply:
changed it back to use local.service as default